Sheragi fighters and minor rebalancing (#5444)
endless-sky/endless-sky: data/sheragi/archaeology missions.txt

				`    (Yes.)`
				`    (No.)`
					defer
			`	You give Albert a call, and arrange to meet with him to plan the next step. He shows up at your ship, greets you, and starts loading in some equipment, along with the jump drive you brought. He has assembled a crew of around 150 people, including a specialized team of shipyard workers, a large amount of veteran crew members, himself, and the three Hai archaeologists, who are comfortable to let go of their disguise once they're back in your ship.`

> 			`	"<first>, meet the people who will help us uncover the fate of the Emerald Sword. I've just briefed them on the situation and what we plan to do. We're going to take some specialized tools with us, along with the jump drive, which should allow us to repair and outfit the ship in any world we can land on. This might look like a lot of people, but if we want to recover a derelict ship, I suspect it will be no easy task. If you're ready to go, here are the coordinates to Zubenelhakrabi."`

Reviewer: ```suggestion
			`	"<first>, meet the people who will help us uncover the fate of the Emerald Sword. I've just briefed them on the situation and what we plan to do. Along with the jump drive, we're going to take some specialized tools with us, which should allow us to repair and outfit the ship in any world we can land on. This might look like a lot of people, but if we want to recover a derelict ship, I suspect we'll need just this many. If you're ready to go, here are the coordinates to Zubenelhakrabi."`
```
Putting `along with the jump drive` immediately before the following phrase implies the jump drive is the device which will allow them to `repair and outfit the ship in any world [they] can land on`. Suggested the other bit because it more directly answers the concern that 150 people `might look like a lot`.

			`	For most of the next day, the workers unload some outfitting and repairing equipment, while the spare crew take turns thinking of names for each fighter, and scientists slowly work out the functions of the ship, briefing you and your crew on how to operate it. With the Jump Drive installed, all that is left is to return the ship to human space for further work.`

mission "Sheragi Archaeology: give ship 2"
	landing
	invisible
	source "Echoes of Tragedy"
	destination "Zug"
	to offer
		has "Sheragi Archaeology: The Emerald Sword 3: done"
	on offer

> 		give ship "Demonheart" "Discount Flak Round"

Reviewer 2: Umm... you can give multiple ships in a single mission.

Author: I should've made a bug report about this.
The give ship command apparently doesn't work to give multiple ships of the same type.
When i attempted to give all 7 ships on the same mission, only the emerald sword, and the last fighter were given.
When i attempted to give the emerald sword and a fighter on the first mission, and more fighters on different missions, the first fighter that was supposed to be given along with the emerald sword wasn't given.
This was the workaround i ended up with, but there is something wrong with the code.

Author: I've now submitted a detailed bug report on #5447

			`	After many long hours walking around, the lights of the ship finally flicker to life, along with sirens bringing attention to every other damaged subsystem, startling everyone. One of the specialized crew members comes to you, explaining that they finally managed to turn on the power plant and fix enough of the ship to move it around, but that they need to land it in order to bring in the heavier equipment and fix the rest of it.`

mission "Sheragi Archaeology: Epilogue"
	landing
	name "The Emerald Sword"
	description "Return to Zug."
	source "Valley of the Damned"
	destination "Zug"
	cargo "Outfitting Equipment" 25
	to offer
		has "Sheragi Archaeology: The Emerald Sword 3: done"
	on offer
		give ship "Emerald Sword (Fixed)" "Emerald Sword"
		give ship "Black Diamond" "Discount Flak Round"
		give ship "Black Diamond" "How Do I Steer"
		give ship "Black Diamond" "An Implicit Comparison To Fleas"
		give ship "Black Diamond" "Punch Biopsy"
		give ship "Black Diamond" "Just The Letter H"

> 		give ship "Black Diamond" "I Got The Bees You Need"

Author: People seemed to like these, but yeah, i'm still looking for names. I think leaving it empty would make it less... significant?

> 		give ship "Black Diamond" "I Got The Bees You Need"

Reviewer 3: "Stormheart"
"Onyx Chariot"
"Firebrand"
"Frostherald"
"Ivory Talon"
"Earthshatter"

> 		give ship "Black Diamond" "I Got The Bees You Need"

Reviewer 4: "Three Skeleton Key"

> 		give ship "Black Diamond" "I Got The Bees You Need"

Author: These are pretty good, thank you!

		conversation
			`As you enter the atmosphere of the alien world, you begin running some scans of the air quality to determine if it is safe to breathe. You quickly learn why it is that the Sheragi perished in space; the atmosphere has a high concentration of chlorine gas, fatal to all living creatures aside from those that seem to have miraculously evolved here. It takes some time to find a suitable landing spot - high enough to escape the highest concentrations of chlorine gas in the atmosphere, but flat enough to land on. As luck would have it, the shipboard scanner reveal anomalous structures over a mesa, and as you come in for a landing, it becomes apparent that these are the ruins of a large settlement. The environmental suits brought along "just in case" turn out to be very useful here. The atmosphere would be unhealthy to breathe for long periods of time, and very uncomfortable in the short term`

> 			`	While the crew starts to work on the Emerald Sword, you and the archaeologists head into the settlement, documenting what they see. The buildings were all connected to each other to avoid the exterior atmosphere, although nowadays, not much remains. They were made of glass, scrap, and a kind of yellowish white plastic, the same plastic that the carapaces of animals here are made of. They were melted into shape and built into walls over time, giving the buildings a rounded, anachronistic look, like mud huts made of modern materials. All around the inside there are curtains and rags hanging from the ceiling to the ground. Closer inspection reveal those rags to have traces of chemicals, likely made to react with the chlorine in the air to keep the atmosphere inside breathable.`

Reviewer 2: ```suggestion
			`	While the crew starts to work on the Emerald Sword, you and the archaeologists head into the settlement, documenting what you see. The buildings were all connected to each other to avoid the exterior atmosphere, although nowadays, not much remains. They were made of glass, scrap, and a kind of yellowish white plastic, the same plastic that the carapaces of animals here are made of. They were melted into shape and built into walls over time, giving the buildings a rounded, anachronistic look, like mud huts made of modern materials. All around the inside there are curtains and rags hanging from the ceiling to the ground. Closer inspection reveal those rags to have traces of chemicals, likely made to react with the chlorine in the air to keep the atmosphere inside breathable.`
```

> 			`	The smaller rooms you come across appear to be just family homes, with a simplistic, single room interior, while the larger ones had other purposes, like what appears to have been a classroom, an animal carapace storage room, holding the exoskeletons of many different species, a communal kitchen with remains of a solar oven, and most strikingly, a hatchery, with relatively small beds, nests, and makeshift toys strewn about. Most of the rooms are littered with skeletons and remains, from the largest hunter-gatherers to the smallest young ones, a grim scene suggesting that some unknown catastrophe ended their lives all at once. Even then, the Sheragi on the surface apparently survived for much longer than the ones in the Emerald Sword, perhaps for many generations. Had the ship landed, they could have survived to this day. The archaeologists take pictures, samples, and minor artifacts, but leave the place mostly undisturbed in respect for a lost civilization, discussing the possible history of this place.`			

Reviewer 2: Changing "they could have survived" to "they might have survived" to make it less definitive.
```suggestion
			`	The smaller rooms you come across appear to be just family homes, with a simplistic, single room interior, while the larger ones had other purposes, like what appears to have been a classroom, an animal carapace storage room, holding the exoskeletons of many different species, a communal kitchen with remains of a solar oven, and most strikingly, a hatchery, with relatively small beds, nests, and makeshift toys strewn about. Most of the rooms are littered with skeletons and remains, from the largest hunter-gatherers to the smallest young ones, a grim scene suggesting that some unknown catastrophe ended their lives all at once. Even then, the Sheragi on the surface apparently survived for much longer than the ones in the Emerald Sword, perhaps for many generations. Had the ship landed, they might have survived to this day. The archaeologists take pictures, samples, and minor artifacts, but leave the place mostly undisturbed in respect for a lost civilization, discussing the possible history of this place.`			
```

> 			`	For most of the next day, the workers unload some outfitting and repairing equipment, while the spare crew take turns thinking of names for each fighter, and scientists slowly work out the functions of the ship, briefing you and your crew on how to operate it. With the Jump Drive installed, all that is left is to return the ship to human space for further work.`

Reviewer 2: ```suggestion
			`	For most of the next day, the workers unload some outfitting and repairing equipment, while the spare crew take turns thinking of names for each fighter, and scientists slowly work out the functions of the ship, briefing you and your crew on how to operate it. With the jump drive installed, all that is left is to return the ship to human space for further work.`
```

				accept
	on complete
		log `Obtained an ancient Sheragi warship whose name translates to the Emerald Sword. It was crewed by mutineers that fled from their civil war and escaped into interstellar space. Found orbiting a disconnected star, it is an impressively large ship, fit for the equally large Sheragi. The ship's main weapon is a fusion-pumped gamma-ray laser called the 'Dragonflame Cannon.' Despite the relatively primitive outfits, it's still a very capable war machine due to its size.`

> 		payment 5000000

Author: I felt like it was appropriate given that no one knew if the ship would be there, but yes, you're right.

			period 44.5228
			offset 128.448

> planet "Valley of the Damned"

Author: That is just to distance myself from naming everything after songs of one single band. It still fits, in the sense that if you find yourself on a valley there, you're damned. :P

> planet "Valley of the Damned"

Reviewer 2: Well I don't have any issues with multiple references to the same band, and if people like "Echoes of Tragedy" better then I don't see why we'd change it.

> planet "Valley of the Damned"

Reviewer 3: Either name is fine to me, though I'd also be in favor of "Field of Lost Souls."
